HedgeModManager/Resources/Codesv2/SonicFrontiers.hmm

```diff
@@ -4688,6 +4688,26 @@ Patch "Infinite Quick Cyloop Gauge" in "Cheats" by "NM"
     );
 }
 
+Patch "Play Monologues Unconditionally" in "Audio" by "WasifBoomz" does
+/*
+The random voice clips around the island are played no matter what.
+
+Previously, they would not play if Sonic was not on regular terrain.
+*/
```

Comment:
Change this description to the following;

```
Removes restrictions for random monologue playback during gameplay.

Notes;
- This allows monologues to play anywhere, rather than only on heightmap terrain.
```